### ISSUE TITLE
[bug] Generate sql code

### ISSUE THREAD

**Issue by II11II** [serverpod/serverpod#368]
While generating ```user``` protocol, I found the serverpod generator does not avoid using reserved sql keywords

Actual behavior: does not avoid using reserved sql keywords while generating tables (Table is not generated ) 
```
CREATE TABLE user (
  "id" serial,
  "firstName" text NOT NULL,
  "lastName" text,
  "avatar" text NOT NULL
);

ALTER TABLE ONLY user
  ADD CONSTRAINT user_pkey PRIMARY KEY (id);

```
Expected behavior: avoid using reserved sql keywords while generating tables, apply double quotes to table name
```
CREATE TABLE "user" (
  "id" serial,
  "firstName" text NOT NULL,
  "lastName" text,
  "avatar" text NOT NULL
);

ALTER TABLE ONLY "user"
  ADD CONSTRAINT user_pkey PRIMARY KEY (id);

```

**Comment by vlidholt** [comment]
Will fix for the 1.0 release, thanks for catching this!

**Comment by II11II** [comment]
https://github.com/serverpod/serverpod/pull/384
